src/smart-components/OpenchannelConversation/index.tsx

```diff
@@ -306,11 +306,15 @@ export const OpenchannelConversation = (props: Props): JSX.Element => {
       {
         renderMessageInput
           ? (
-            renderMessageInput({
-              channel: currentOpenChannel,
-              user: user,
-              disabled: disabled,
-            })
+            <div className="sendbird-openchannel-footer">
```

Comment:
Is there no design issue after this change?

Reply:
There was issue before 😆

Reply:
Oh okok haha